### ISSUE TITLE
Lição 1 - Atividade 1 

### ISSUE THREAD

**Issue by simas0** [livro-aberto/fracoes_livro_piloto#150]
**Oficina de Frações no Ensino Fundamental 1 - UFF, 13/02/2017**

Atividade 1 c) Preocupação com a divisão exata. Alguns professores queriam medir a barra de chocolate com uma régua e dividir o número por três. Ficaram preocupados com a exatidão. Alguém usou um barbante para medir a barra e usou a maleabilidade do barbante para encontrar o terço.

A divisão por três trouxe dificuldade. Os professores buscavam uma forma de dobrar as folhas de reprodução duas vezes para conseguir três partes iguais. Observe que isso se afasta totalmente da situação real apresentada. A barra não é tão maleável assim.

Alguém sugeriu mudar para divisão em 4 partes como se isso fosse simplificar para o estudante efetuar a divisão utilizando este processo de dobras. Outros disseram que "terço" não é uma palavra tão familiar ao estudante como "meio" ou "quarto".

Professora que utilizou com seus alunos diz que quando se tratava de irmãos a ideia da divisão em partes iguais era mais marcada.

A figura do item c) está errada. Devia ser um retângulo.

**Comment by simas0** [comment]
Observar ao professor que não é importante a precisão na medida. Isso afasta da estratégia utilizada numa situação real em que geralmente se usa uma aproximação visual e não medições. Embora a estratégia do barbante acima seja muito prática, um bom corte no famoso olhômetro não obteria um corte muito diferente.

**Comment by simas0** [comment]
Empurrar as **Recomendações para o desenvolvimento da atividade** para a outra coluna.

**Comment by cydara** [comment]
Não entendi se vamos mudar o desenho do ítem (c) ou se foi um registro da oficina...

**Comment by simas0** [comment]
Sim, precisamos mudar o desenho do item (c) porque para um estudante do 4º ano pode ser desafiador e desviar dos nossos objetivos, que ele particione a figura tridimensional, como pede o exercício. Concorda?
